### DISCUSSION TITLE
How to read and save on floppy disk?

### DISCUSSION THREAD

**Discussion by tymon-k** [fnuecke/oc2#285] [Q&A]
I have a question about how to read or write from a floppy disk using the command.

**Comment by tymon-k** [comment]
I have this question because the usual things in the documentation don't work for me.

**Reply by Shazuli** [reply]
I haven't tried this mod in a hot minute, but have you tried the `mount` command if it exists?

**Comment by TGJek** [comment]
I just partitioning the floppy disk and mount the created partition.
